### DISCUSSION TITLE
TESTERS NEEDED - Comment to be updated of the latest releases

### DISCUSSION THREAD

**Discussion by lacymorrow** [lacymorrow/crossover#330] [General]
I need folks to test pre-release versions for regressions before I push anything live to automatically update.

Anyone interested?

Comment here

**Comment by lacymorrow** [comment]
V4 has been built from scratch, took over a month.
Alpha will be posted today, it should be stable-ish but use at your own risk

**Comment by RebelLion420** [comment]
I will

**Comment by lacymorrow** [comment]
# alpha.1

https://github.com/lacymorrow/crossover/releases/tag/v4.0.0-alpha.1

Please test the app lock/unlock and other settings.

**Comment by ghost** [comment]
For me it not works at all, for starters main window for some reason on top off all the others, and when I use Lock Crosshair hotkeys it start blinking and crash.
https://youtu.be/v-H3DhHEL7E

**Reply by lacymorrow** [reply]
Thank you for the report and video!

**Reply by lacymorrow** [reply]
This is actually the best bug report, much appreciated 🍻 
I've fixed the issue and will try to have another version tonight or tomorrow.

**Reply by lacymorrow** [reply]
Two years later, I have a beta. I'm wondering if you mind seeing if it fixes your bugs.

**Comment by lacymorrow** [comment]
New version of v3:
https://github.com/lacymorrow/crossover/releases/tag/v3.3.7

**Comment by lacymorrow** [comment]
# v3.4.0

> New crosshairs, new system hooks for actions
https://github.com/lacymorrow/crossover/releases/tag/v3.4.0

**Comment by lacymorrow** [comment]
# v4 BETA
https://github.com/lacymorrow/crossover/releases/tag/v4.0.0-beta.1
